### PR TITLE
CENSORED tweaks

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About The Pull Request

- Increases attribute requirements for CENSORED weapon.
- Increases damage dealt with CENSORED weapon.
- Increases damage dealt by CENSORED itself.

## Why It's Good For The Game

- Slightly more accurate to the game. Makes it a weapon that can only be used by strong agents.
- With armor gone, it's only natural it should receive a buff somewhere else.
- CENSORED doesn't have any real special attacks right now, making it really easy to fight. This should *slightly* change it.

### CODE REVIEW

File: code/game/objects/items/ego_weapons/aleph.dm [hunk 1]
```diff
@@ -341,21 +341,21 @@
...
 	hitsound = 'sound/weapons/ego/censored1.ogg'
 	attribute_requirements = list(
-							FORTITUDE_ATTRIBUTE = 80,
-							PRUDENCE_ATTRIBUTE = 100,
-							TEMPERANCE_ATTRIBUTE = 80,
-							JUSTICE_ATTRIBUTE = 80
+							FORTITUDE_ATTRIBUTE = 110,
+							PRUDENCE_ATTRIBUTE = 90,
+							TEMPERANCE_ATTRIBUTE = 90,
+							JUSTICE_ATTRIBUTE = 90
```
Comment (dsmith328) [inline code/game/objects/items/ego_weapons/aleph.dm:354]: Why the increased requirements? This is essentially on par with most of our Aleph weapons at the moment now, is it not?

Comment (Kitsunemitsu) [reply code/game/objects/items/ego_weapons/aleph.dm:354]: This is a strange attribute requirement.

**Comment by Kitsunemitsu** [comment]
Egor, this is honestly too much.
CENSORED is already a pretty great weapon and out-DPSes Smile before the buff, with regular attacks

**Comment by dsmith328** [comment]
> Why buff the CENSORED ego even more? Was the 300 death laser + 40% resistance across the board not enough?

It doesn't have the resistances anymore

**Comment by dsmith328** [comment]
Honestly this could be a normal 100 cost if we get the armor with it's current stats. I don't mind it getting more damage overall and the special. Just makes it a really offensive Aleph weapon with no other benefits. Smile is still really good for the EXECUTE so this doesn't really steal any spotlights.

**Comment by EgorDinamit** [comment]
**If** or **When** we do get the armor, it can be adjusted.
It's a 250 PE weapon from one of the most difficult abnormalities to work on, it **deserves** to be special, just like paradise lost.

**Pull request merged**